kernel/src/utilities/math.rs

```diff
@@ -115,7 +115,7 @@ fn ln_1to2_series_approximation(x: f32) -> f32 {
     //https://en.wikipedia.org/wiki/Remez_algorithm
     let ln_1to2_polynomial: f32 = -1.741_793_9_f32
         + (2.821_202_6_f32
-            + (-1.469_956_8_f32 + (0.447_179_55_f32 - 0.056_570_851_f32 * x_working) * x_working)
+            + (-1.469_956_8_f32 + (0.447_179_55_f32 - 0.056_570_85_f32 * x_working) * x_working)
```

Comment:
Did this just trim a bit off the constant here?

Reply:
That final `1` is beyond the precision of a 32-bit float. Your options are 0.05657085**0**3127 or 0.05657085**4**038. It looks like there are a couple clippy lints that look at floating point literals to ensure that they make sense.

Reply:
On the one hand, that makes sense — on the other hand, curious what it was doing before... always wary about changing magic constants, even in the insignificant bits.

@alexandruradovici , looks like you added this in 8daa35 — any issues here?

Reply:
As I understand it this is the same as before, Rust was truncating the one.

https://rust-lang.github.io/rust-clippy/master/index.html#/excessive_precision